pkg/plugin/pr-sanitizer/cmd/main.go

```diff
@@ -3,29 +3,18 @@ package main
 import (
 	"k8s.io/test-infra/prow/pluginhelp"
 
-	"github.com/arquillian/ike-prow-plugins/pkg/github"
 	"github.com/arquillian/ike-prow-plugins/pkg/github/client"
-	"github.com/arquillian/ike-prow-plugins/pkg/log"
 	pluginBootstrap "github.com/arquillian/ike-prow-plugins/pkg/plugin"
+	prsanitizer "github.com/arquillian/ike-prow-plugins/pkg/plugin/pr-sanitizer"
```

Comment:
How about `prSanitizer` instead of  `prsanitizer`?

Reply:
Removing it as it is not required at all.